frontend/test/metabase/scenarios/admin/databases/add.cy.spec.js

```diff
@@ -184,18 +184,43 @@ describe("scenarios > admin > databases > add", () => {
     );
   });
 
+  it.skip("should respect users' decision to manually sync large database (metabase#17450)", () => {
+    const H2_CONNECTION_STRING =
+      "zip:./target/uberjar/metabase.jar!/sample-dataset.db;USER=GUEST;PASSWORD=guest";
```

Comment:
Is the CWD really not the same as the jar file? Interesting. Could be true on CI, just strange.